backend/editor/entries.py

```diff
@@ -126,11 +126,20 @@ async def get_and_parse_taxonomy(self, uploadfile: UploadFile | None = None):
                 )
                 await run_in_threadpool(self.parse_taxonomy, filepath)
                 async with TransactionCtx():
-                    await edit_project(self.project_name, ProjectEdit(status=ProjectStatus.OPEN))
+                    error_node = await get_error_node(self.project_name)
+                    errors_count = len(error_node.errors) if error_node else 0
+                    await edit_project(
+                        self.project_name,
+                        ProjectEdit(status=ProjectStatus.OPEN, errors_count=errors_count),
```

Comment:
Is this better than getting errors in a cipher query.

In the future I was imagining that appart parse errors, we would have logical errors that may change as you edit taxonomy.

Reply:
I'm not sure: it seems that form the current use of errors, this seems to be cleaner and more performant
But if we add logical errors to the mix in the future that can be added when we edit a node, this may be not true..

Reply:
It was only a question, I think we can leave it like it :-)